app/controllers/user.rb

```diff
@@ -0,0 +1,48 @@
+module ChitterModule
+
+  module Routes
+
+    class UserController < Base
+
+      get '/users/new' do
+        erb :'users/new'
+      end
+
+      post '/users' do
+        @user = User.create(email: params[:email],
+                           name: params[:name],
+                           user_name: params[:user_name],
+                           password: params[:password],
+                           password_confirmation: params[:password_confirmation])
```

Comment:
Line is too long. [81/80]<br>Align the elements of a hash literal if they span more than one line.